src/lib/lint.ts

```diff
@@ -42,9 +44,61 @@ export function parseOutdatedSchoolPeriods(desc: string): string[] {
     .filter((s) => !s.includes(THIS_YEAR.toString()));
 }
 
+/** Parses the given description for ethnicities and returns matches. */
+export function parseGradeLevels(desc: string): GradeLevel[] {
+  const lowerDesc = desc.toLowerCase();
+  const keywords = {
+    [GradeLevel.MiddleSchool]: ['middle school', '[78]th grade'],
+    [GradeLevel.HsFreshman]: ['9th grade'],
+    [GradeLevel.HsSophomore]: ['10th grade'],
```

Comment:
what about including `high school freshman` & `high school sophomore`?

Reply:
I left them out because in our scholarship sheet none of the references to freshmen and sophomores referred to high school. Also I remembered yesterday that some school systems have middle school go through 9th grade while high school starts at 10th so it's probably for the better. If we find cases where we're incorrectly assuming college freshmen/sophmores we can revisit this. This is after all just a heuristic and is not meant to block editing anything.